templates/files/conf/k8s-addons

```diff
@@ -128,6 +127,23 @@ $KUBECTL -n kube-system delete deploy calico-kube-controllers --ignore-not-found
 # delete legacy nginx daemonset
 $KUBECTL -n kube-system delete daemonset nginx-ingress-controller --ignore-not-found=true
 
+# delete nginx ingress controller installed by terraform <= 5.10
+$KUBECTL -n kube-system delete deploy default-http-backend --ignore-not-found=true
+$KUBECTL -n kube-system delete svc default-http-backend --ignore-not-found=true
+$KUBECTL -n kube-system delete cm ingress-nginx --ignore-not-found=true
+$KUBECTL -n kube-system delete serviceaccount nginx-ingress-controller --ignore-not-found=true
+$KUBECTL -n kube-system delete deploy nginx-ingress-controller --ignore-not-found=true
+$KUBECTL -n kube-system delete cm ingress-nginx --ignore-not-found=true
+$KUBECTL -n kube-system delete svc nginx-ingress-controller --ignore-not-found=true
+$KUBECTL delete clusterrolebinding nginx-ingress-controller --ignore-not-found=true
+$KUBECTL delete clusterrole nginx-ingress-controller --ignore-not-found=true
+$KUBECTL -n kube-system delete rolebinding nginx-ingress-controller --ignore-not-found=true
+$KUBECTL -n kube-system delete role nginx-ingress-role --ignore-not-found=true
```

Comment:
the naming is different from the app?

Reply:
otherwise same comment as with coredns

Reply:
yeah see my other comment. There is no overlap, but nothing ensures it'll never happen.
The goal should be to remove those delete commands from a future release, as soon as all MCs will be upgraded to this release.
Open to hear your opinions about